db/migrate/20200729191640_drop_person_id_from_moves.rb

```diff
@@ -0,0 +1,6 @@
+class DropPersonIdFromMoves < ActiveRecord::Migration[6.0]
+  def change
+    remove_foreign_key :moves, column: :person_id, name: 'fk_rails_moves_person_id', to_table: :people
```

Comment:
Unfortunately the `remove_reference` wrapper for `remove_foreign_key`, `remove_index` and `remove_column` doesn't let you pass options to the remove_column method meaning we can't have rollback compatibility/set the type to `:uuid`.